### ISSUE TITLE
computational problem

### ISSUE THREAD

**Issue by George191** [pythonprofilers/memory_profiler#251]
-    121                                             # disk = psutil.disk_usage('/').percent
   122                                             # cpu = psutil.cpu_percent(0)
   123                                             # cpu_kernel = psutil.cpu_percent(percpu=True)
   124
   125 1658.176 MiB    0.000 MiB                   args = {k: v.to(device) for k, v in batch.items() if k != 'label'}
   126 1892.559 MiB  263.688 MiB                   logits = model(**args)
   127 1892.559 MiB    0.102 MiB                   loss = F.cross_entropy(logits, batch['label'].to(device))
   128 1658.176 MiB    0.000 MiB                   loss.backward()
   129 1658.176 MiB  127.637 MiB                   optimizer.step()
   130 1658.176 MiB    0.008 MiB                   model.zero_grad()
   131 1658.176 MiB    0.000 MiB                   total_memory += memory
   132 1658.176 MiB    0.000 MiB                   pbar.set_description(f'epoch: {epoch} loss: {loss.item():.8f} The Memory Usage: {memory}'
   133                                                                  f' Total Memory Usage: {total_memory}')
   134

**Comment by George191** [comment]
The increase is large, the consumption has not changed.
